core/services/gateway/handlers/functions/handler.functions.go

```diff
@@ -82,8 +89,21 @@ func NewFunctionsHandlerFromConfig(handlerConfig json.RawMessage, donConfig *con
 			return nil, err
 		}
 	}
+	var subscriptions OnchainSubscriptions
+	if cfg.OnchainSubscriptions != nil {
+		chain, err2 := legacyChains.Get(cfg.OnchainSubscriptionsChainID)
+		if err2 != nil {
+			return nil, err2
+		}
+		subscriptions, err2 = NewOnchainSubscriptions(chain.Client(), *cfg.OnchainSubscriptions, lggr)
+		if err2 != nil {
+			return nil, err2
+		}
+	}
+	minimumBalanceWei := new(big.Int)
+	big.NewFloat(0).Mul(big.NewFloat(cfg.MinimumSubscriptionBalanceLink), big.NewFloat(1e18)).Int(minimumBalanceWei)
```

Comment:
Maybe better to have Juels in config? I'm worried about precision.

Reply:
changed to what we use in other cases like this: assets.Link.